tests/test_base_group.py

```diff
@@ -1,5 +1,22 @@
-import logging
-import random
+# This file is part of ts_standardscripts
```

Comment:
Same copy and paste. Please change to ts_observatory_control and perhaps check other files as well while you're at it.